templates.tpl

```diff
@@ -0,0 +1,136 @@
+  },
+  "description": "PBX Plus is an advanced communications system and IVR for your business needs.\nhttps://infinity.500apps.com/ PBXPlus",
```

Comment:
PBXPlus

Reply:
why there is a space
<img width="275" alt="Screenshot 2021-12-08 at 8 05 36 AM" src="https://user-images.githubusercontent.com/66065056/145137640-fe775905-a9d9-491f-9d4f-542c98455583.png">

Reply:
resolved